andela/wg-shleemypants
# Ft member comparison

**What does this PR do?**
Enables two users training together to have weight data comparison

**Description of tasks to be completed**

- Define endpoints to fetch weight data for the currently logged in user and the user in comparison.
- Implement the necessary Js to plot the graphs.
- Create a template to display the two graphs side by side.
- Have a dropdown to select users to compare with
- Include a comparison tab in the navigation bar

**How can this be tested manually?**

- Navigate to the comparison tab, this should have a graph with the currently logged in user, select another user and their graph will appear on the right side.

**Relevant tracker stories**

- 157110622

**Additional notes**
- this feature requires javascript hence ensure all the static files are loading as expected.

**Screenshots**
Default comparison page

<img width="1197" alt="screen shot 2018-05-15 at 2 53 20 pm" src="https://user-images.githubusercontent.com/17777137/40055211-d160bcca-584f-11e8-92d7-41b9f654cffe.png">

Drop down to select a comparison user 

<img width="1192" alt="screen shot 2018-05-15 at 2 56 14 pm" src="https://user-images.githubusercontent.com/17777137/40055365-57c1f964-5850-11e8-8fcd-f5569346c0c6.png">


A comparison with another user

<img width="1198" alt="screen shot 2018-05-15 at 8 07 39 am" src="https://user-images.githubusercontent.com/17777137/40055624-3d40842e-5851-11e8-8876-0b391943a42d.png">

Comparison with another different user

<img width="1200" alt="screen shot 2018-05-15 at 8 08 16 am" src="https://user-images.githubusercontent.com/17777137/40055678-6cbcc4c4-5851-11e8-964d-8501119e7ce3.png">
